### PR TITLE
Spells/Generic: Fix Mag'har Orc racial spell

### PR DESCRIPTION
**Pull request by aquadeus**
**Changes proposed:**

-   Fix Mag'har Orc racial spell

**Tests performed:**

Tested in-game

### CODE REVIEW

File: src/server/scripts/Spells/spell_generic.cpp [hunk 1]
```diff
@@ -5032,6 +5032,55 @@ class spell_gen_mount_check_aura : public AuraScript
...
+            SPELL_ZEAL_OF_THE_BURNING_BLADE,
+            SPELL_FEROCITY_OF_THE_FROSTWOLF,
+            SPELL_MIGHT_OF_THE_BLACKROCK
+        });
+    }
+
+    void Oncast()
+    {
+        Unit* caster = GetCaster();
+        if (!caster)
```
Comment (mdX7) [inline src/server/scripts/Spells/spell_generic.cpp:5062]: caster never null in spellscript

File: src/server/scripts/Spells/spell_generic.cpp [hunk 2]
```diff
@@ -5032,6 +5032,55 @@ class spell_gen_mount_check_aura : public AuraScript
...
+        return ValidateSpellInfo(
+        {
+            SPELL_RICTUS_OF_THE_LAUGHING_SKULL,
+            SPELL_ZEAL_OF_THE_BURNING_BLADE,
+            SPELL_FEROCITY_OF_THE_FROSTWOLF,
+            SPELL_MIGHT_OF_THE_BLACKROCK
+        });
+    }
+
+    void Oncast()
```
Comment (mdX7) [inline src/server/scripts/Spells/spell_generic.cpp:5059]: UpperCamelCase

File: sql/updates/world/master/2022_99_99_99_world_maghtar_orc_racial_spell.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` = 274738;
```
Comment (mdX7) [inline sql/updates/world/master/2022_99_99_99_world_maghtar_orc_racial_spell.sql:1]: delete by ScriptName

Comment (aquadeus) [reply sql/updates/world/master/2022_99_99_99_world_maghtar_orc_racial_spell.sql]: ok done

**Pull request merged**